src/coreclr/vm/methodtable.cpp

```diff
@@ -2804,18 +2804,12 @@ bool MethodTable::IsLoongArch64OnlyOneField(MethodTable * pMT)
 {
```

Comment:
Could you please move this method outside the `TARGET_LOONGARCH64` ifdef, rename it to `MethodTable::IsOnlyOneField`, and use it for both the LoongArch and RISCV (delete `IsRiscV64OnlyOneField` method that is identical).

Reply:
OK，I have already updated this, thanks very much!

Reply:
@tomeksowi @clamp03 @sirntar ,this also related to RISCV, could you please also take a look?